pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -22,6 +22,7 @@ import (
 	"k8s.io/kubernetes/pkg/apis/settings"
 	"k8s.io/kubernetes/pkg/apis/storage"
 	"k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy"
+	"k8s.io/metrics/pkg/apis/metrics"
```

Comment:
You can't depend on this.

Reply:
ok, I can just type out the name instead.  Was trying to follow what seemed to be the convention of using GroupName from the actual API package.